src/components/index/RoadMapSection.tsx

```diff
@@ -0,0 +1,154 @@
+import { Container } from "@components/commons/Container";
+import { SectionTitle } from "@components/commons/SectionTitle";
+import { LeftArrow } from "@components/icons/assets/roadmap/LeftArrow";
+import { RightArrow } from "@components/icons/assets/roadmap/RightArrow";
+import classNames from "classnames";
+import { useTranslation } from "next-i18next";
+import { CSSProperties, ReactEventHandler } from "react";
+import Slider from "react-slick";
+
+export function YearAheadRoadMapSection(): JSX.Element {
+  const { t } = useTranslation("page-index");
+
+  return (
+    <>
+      <Container>
+        <SectionTitle
+          text={t("YearAheadRoadMapSection.label")}
+          customStyle="lg:w-fit md:w-[409px] w-[272px]"
```

Comment:
is it possible to use w-fit for md and mobile as well ? theres a color difference in the linear gradient for mobile and tablet

Reply:
updated